.github/workflows/stalebot.yml

```diff
@@ -0,0 +1,24 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 9 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need to generate some token or it happens automatically? :thinking:

Reply:
No, GH will take care of it.